include/bluetooth/audio/audio.h

```diff
@@ -1,9 +1,1699 @@
+
+	/** @brief Stream QoS set callback
+	 *
+	 *  QoS set callback is called whenever an Audio Stream Quality of
+	 *  Service has been set or updated.
+	 *
+	 *  @param stream Stream object that had its QoS updated.
+	 */
+	void (*qos_set)(struct bt_audio_stream *stream);
+
+	/** @brief Stream enabled callback
+	 *
+	 *  Enabled callback is called whenever an Audio Stream has been
+	 *  enabled.
+	 *
+	 *  @param stream Stream object that has been enabled.
+	 */
+	void (*enabled)(struct bt_audio_stream *stream);
+
+	/** @brief Stream started callback
+	 *
+	 *  Started callback is called whenever an Audio Stream has been started
+	 *  and will be usable for streaming.
+	 *
+	 *  @param stream Stream object that has been started.
+	 */
+	void (*started)(struct bt_audio_stream *stream);
+
+	/** @brief Stream metadata updated callback
+	 *
+	 *  Metadata Updated callback is called whenever an Audio Stream's
+	 *  metadata has been updated.
+	 *
+	 *  @param stream Stream object that had its metadata updated.
+	 */
+	void (*metadata_updated)(struct bt_audio_stream *stream);
+
+	/** @brief Stream disabled callback
+	 *
+	 *  Disabled callback is called whenever an Audio Stream has been
+	 *  disabled.
+	 *
+	 *  @param stream Stream object that has been disabled.
+	 */
+	void (*disabled)(struct bt_audio_stream *stream);
+
+	/** @brief Stream stopped callback
+	 *
+	 *  Stopped callback is called whenever an Audio Stream has been
+	 *  stopped.
+	 *
+	 *  @param stream Stream object that has been stopped.
+	 */
+	void (*stopped)(struct bt_audio_stream *stream);
+
+	/** @brief Stream released callback
+	 *
+	 *  Released callback is called whenever a Audio Stream has been
+	 *  released and can be deallocated.
+	 *
+	 *  @param stream Stream object that has been released.
+	 */
+	void (*released)(struct bt_audio_stream *stream);
+
+	/** @brief Stream connected callback
+	 *
+	 *  If this callback is provided it will be called when the
+	 *  isochronous stream is connected.
+	 *
+	 *  @param stream The stream that has been connected
+	 */
+	void (*connected)(struct bt_audio_stream *stream);
+
+	/** @brief Stream disconnected callback
+	 *
+	 *  If this callback is provided it will be called when the
+	 *  isochronous stream is disconnected, including when a connection gets
+	 *  rejected.
+	 *
+	 *  @param stream The stream that has been Disconnected
+	 *  @param reason HCI reason for the disconnection.
+	 */
+	void (*disconnected)(struct bt_audio_stream *stream, uint8_t reason);
+
+	/** @brief Stream audio HCI receive callback.
+	 *
+	 *  This callback is only used if the ISO data path is HCI.
+	 *
+	 *  @param stream Stream object.
+	 *  @param buf  Buffer containing incoming audio data.
+	 */
+	void (*recv)(struct bt_audio_stream *stream, struct net_buf *buf);
+};
+
+/** @brief Audio Capability type */
+enum bt_audio_pac_type {
+	BT_AUDIO_SINK = 0x01,
+	BT_AUDIO_SOURCE = 0x02,
+} __packed;
+
+#define BT_AUDIO_CONTENT_UNSPECIFIED     BIT(0)
```

Comment:
I guess these should _CONTEXT_, not _CONTENT_

Also, these overlap with the codec defines in line 123 - 189 - do we need both?
(Or are they actually separate entities in the spec?)

Reply:
I can't find them in any specification, so I decided to remove them, as we have the context values instead.

Reply:
There has been a lot of confusion between "content" and "context" - "context" is the correct one.
(I believe there are no content values.)